trustification/trustify
# Serve UI in trustify

@bobmcwhirter you mentioned the UI being served by the trustd server. Here is a working example of it. 
- The current PR allows the UI to be served by trustd but the UI can still live alone without the need of trustd (e.g a container of only the UI can be created). So the UI can live in standalone mode but also within trustd.
- This PR implies moving back https://github.com/trustification/trustify-ui to this repository. I don't know if that is a good idea, in the past there were complains about it therefore the UI was moved to its own repository. What is your opinion?

I opened this PR mainly to show your idea in action. I'll defer the decision of where the UI code should live. So no need to merge this PR until we are happy with it.

### How to test this PR
- Checkout this PR
- Make sure you have NodeJS 20 installed and npm >9.5.0
- Execute:
```
cargo run --bin trustd -- --auth-disabled
```

Open http://localhost:8080
